changelog.d/20240520_113001_danfuchs_notebook_directories.md

```diff
@@ -0,0 +1,10 @@
+<!-- Delete the sections that don't apply -->
+
+### Backwards-incompatible changes
+
+- NotebookRunner business now runs all notebooks in a repo by default except in the `mobu_exclude` or its descendants
```

Comment:
You dropped the `mobu_exclude` part. We've also been putting periods at the end of change log entries (sort of an arbitrary style thing).